packages/react/src/components/modal/dialog/styled.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { Button } from '../../buttons/button';
+import { Heading } from '../../heading/heading';
+import { Icon } from '../../icon/icon';
+import { MobileDeviceContextProps } from '../types';
+import { DialogType } from './types';
+
+export const Subtitle = styled(Heading)`
+`;
+
+export const ButtonContainer = styled.div<MobileDeviceContextProps>`
+    display: flex;
+    flex-direction: ${({ isMobile }) => (isMobile ? 'column' : 'unset')};
+    gap: var(--spacing-1x);
+    justify-content: flex-end;
+`;
+
+export const ConfirmButton = styled(Button)`
+`;
+
+export const CancelButton = styled(Button)`
+`;
+
+export const TitleIcon = styled(Icon)<{ $dialogType: DialogType }>`
+    ${({ $dialogType, theme }) => ($dialogType === 'alert' ? `color: ${theme.component['modal-dialog-alert-icon-color']}` : '')};
```

Comment:
Le style est plus visible de meme. Et en retournant null, il va pas output ce style.

```suggestion
    color: ${({ $dialogType, theme }) => $dialogType === 'alert' ? theme.component['modal-dialog-alert-icon-color'] : null};
```